classes/order/Order.php

```diff
@@ -311,6 +311,10 @@ public function getFields()
     public function add($autodate = true, $null_values = true)
     {
         if (parent::add($autodate, $null_values)) {
+            $cart = new Cart((int) $this->id_cart);
```

Comment:
I'm not sure exactly when the Order reference is created and saved Maybe it would be safer to perform this in the update method as well?